custom/icds_reports/management/commands/citus_migration.py

```diff
@@ -0,0 +1,254 @@
+from __future__ import absolute_import, print_function
+from __future__ import unicode_literals
+
+import inspect
+import logging
+from collections import defaultdict
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from sqlalchemy import inspect as sqlinspect
+
+from corehq.apps.userreports.models import StaticDataSourceConfiguration
+from corehq.apps.userreports.util import get_indicator_adapter, UCR_TABLE_PREFIX
+from corehq.sql_db.connections import connection_manager
+from custom.icds_reports.const import AGG_LS_AWC_VISIT_TABLE, AGG_LS_VHND_TABLE, AGG_LS_BENEFICIARY_TABLE, \
+    AGG_GROWTH_MONITORING_TABLE, AGG_DAILY_FEEDING_TABLE, AGG_COMP_FEEDING_TABLE, AGG_CCS_RECORD_CF_TABLE, \
+    AGG_CHILD_HEALTH_THR_TABLE, AGG_CCS_RECORD_THR_TABLE, AGG_CHILD_HEALTH_PNC_TABLE, AGG_CCS_RECORD_PNC_TABLE, \
+    AGG_CCS_RECORD_DELIVERY_TABLE, AGG_CCS_RECORD_BP_TABLE, AGG_INFRASTRUCTURE_TABLE, AWW_INCENTIVE_TABLE
+from custom.icds_reports.models import AggregateInactiveAWW, AggregateSQLProfile, ICDSAuditEntryRecord
+from custom.icds_reports.models.helper import IcdsFile
+
+logger = logging.getLogger(__name__)
+
+IGNORE_TABLES = {
+    'django_migrations',
+    AggregateSQLProfile._meta.db_table,
+    'ucr_table_name_mapping',
```

Comment:
were we able to finally kill all usages of this in code? may be worth killing the model and dropping the table

Reply:
it's still used by `update_child_health_monthly` but that's the only usage.